.golangci.yml

```diff
@@ -109,6 +110,15 @@ linters-settings:
   gosec:
     excludes:
       - G107 # Url provided to HTTP request as taint input https://securego.io/docs/rules/g107
+  importas:
+    # Do not allow unaliased imports of aliased packages.
+    no-unaliased: false
+    # Do not allow non-required aliases.
+    no-extra-aliases: false
+    # List of aliases
+    alias:
+      - pkg: github.com/ava-labs/avalanchego/utils/math
+        alias: safemath
```

Comment:
it's funny that linter didn't complain about the following camino files  that do not follow this `importas` rule (introduced by `97250954107b75977bcd1d96aca3d07cc3e19d61`):
```
Targets
    Occurrences of '"github.com/ava-labs/avalanchego/utils/math"' in Project with mask '*camino*'
Found Occurrences in Project with mask '*camino*'  (15 usages found)
    Production  (15 usages found)
        caminogo  (15 usages found)
            genesis  (3 usages found)
                camino_config.go  (1 usage found)
                    15 "github.com/ava-labs/avalanchego/utils/math"
                camino_genesis.go  (1 usage found)
                    18 "github.com/ava-labs/avalanchego/utils/math"
                camino_unparsed_config.go  (1 usage found)
                    13 "github.com/ava-labs/avalanchego/utils/math"
            utils/json  (1 usage found)
                camino.go  (1 usage found)
                    7 "github.com/ava-labs/avalanchego/utils/math"
            vms/platformvm  (1 usage found)
                camino_service.go  (1 usage found)
                    23 "github.com/ava-labs/avalanchego/utils/math"
            vms/platformvm/api  (1 usage found)
                camino.go  (1 usage found)
                    13 "github.com/ava-labs/avalanchego/utils/math"
            vms/platformvm/dac  (1 usage found)
                camino_fee_distribution_proposal.go  (1 usage found)
                    15 "github.com/ava-labs/avalanchego/utils/math"
            vms/platformvm/deposit  (1 usage found)
                camino_deposit.go  (1 usage found)
                    11 "github.com/ava-labs/avalanchego/utils/math"
            vms/platformvm/state  (1 usage found)
                camino.go  (1 usage found)
                    20 "github.com/ava-labs/avalanchego/utils/math"
            vms/platformvm/txs  (4 usages found)
                camino_add_proposal_tx.go  (1 usage found)
                    12 "github.com/ava-labs/avalanchego/utils/math"
                camino_add_validator_tx.go  (1 usage found)
                    12 "github.com/ava-labs/avalanchego/utils/math"
                camino_deposit_tx.go  (1 usage found)
                    13 "github.com/ava-labs/avalanchego/utils/math"
                camino_rewards_import_tx.go  (1 usage found)
                    11 "github.com/ava-labs/avalanchego/utils/math"
            vms/platformvm/txs/executor  (1 usage found)
                camino_tx_executor.go  (1 usage found)
                    19 "github.com/ava-labs/avalanchego/utils/math"
            vms/platformvm/utxo  (1 usage found)
                camino_locked.go  (1 usage found)
                    16 "github.com/ava-labs/avalanchego/utils/math"
```
imo, we should check that and address it within post merge fixes

Reply:
Yeah.. Actually `"github.com/ava-labs/avalanchego/utils/math"` is imported in 81 file, only small part of them are ours. I hope avax will take care of this linter later, so we won't have to deal with it and create more diff :P